docs/src/neuralestimators_amorized.md

```diff
@@ -0,0 +1,261 @@
+
+Below, we demonstrate how to estimate parameters of the LCA model using the [NeuralEstimators.jl](https://github.com/msainsburydale/NeuralEstimators) package.
+
+## Example
+
+We'll estimate parameters of the LCA model, which is particularly challenging due to its complex dynamics where parameters like leak rate (λ) and lateral inhibition (β) can be difficult to recover.
+
+## Load Packages
+
+```julia
+using NeuralEstimators
+using SequentialSamplingModels
+using Flux
+using Distributions
+using Random
+
+using Plots
+
+Random.seed!(123)
+```
+
+## Define Parameter Bounds
+
+```julia
+# Define parameter bounds for the LCA model
+const ν_min, ν_max = 0.1, 4.0      # Drift rates
+const α_min, α_max = 0.5, 3.5      # Threshold
+const β_min, β_max = 0.0, 0.5      # Lateral inhibition
+const λ_min, λ_max = 0.0, 0.5      # Leak rate
+const τ_min, τ_max = 0.1, 0.5      # Non-decision time
+```
+
+## Define Parameter Sampling
+
+Unlike traditional Bayesian approaches, simulation based inference methods require us to define a prior sampling function to generate training data. We will use this function to sample a range of parameters for training:
+
+```julia
+# Function to sample parameters from priors
+function sample(K::Integer)
+    ν1 = rand(Gamma(2, 1/1.2), K)  # Drift rate 1
+    ν2 = rand(Gamma(2, 1/1.2), K)  # Drift rate 2
+    α = rand(Gamma(10, 1/6), K)    # Threshold
+    β = rand(Beta(1, 5), K)        # Lateral inhibition
+    λ = rand(Beta(1, 5), K)        # Leak rate
+    τ = rand(Gamma(1.5, 1/5.0), K) # Non-decision time
+    
+    # Stack parameters into a matrix (d×K)
+    θ = vcat(ν1', ν2', α', β', λ', τ')
+    
+    return θ
+end
+```
+
+## Define Data Simulator
+
+Neural estimators learn the mapping from data to parameters through simulation. Here we define a function to simulate LCA model data. To do so we will use the LCA function from SequentialSamplingModels.
+
+```julia
+# Function to simulate data from the LCA model
+function simulate(θ, n_trials_per_param)
+    # Simulate data for each parameter vector
+    simulated_data = map(eachcol(θ)) do param
+        # Extract parameters for this model
+        ν1, ν2, α, β, λ, τ = param
+        ν = [ν1, ν2]  # Two-choice LCA
+        σ = 1.0        # Fixed diffusion noise
+        
+        # Create LCA model
+        model = LCA(; ν, α, β, λ, τ, σ)
+        
+        # Generate choices and reaction times
+        choices, rts = rand(model, n_trials_per_param)
+        
+        # Return as a transpose matrix where each column is a trial 
+        return Float32.([choices rts]')
```

Comment:
If you use `f0` to use `Float32` in your `sample`, I think you can get rid of this conversion. If that does not work, open an issue, and I can try to make it more generic.